src/main/app/forms/models/organisationDetails.ts

```diff
@@ -1,9 +1,11 @@
 import { PartyDetails } from './partyDetails'
 import { PartyType } from 'common/partyType'
 import { MaxLength } from '@hmcts/cmc-validators'
+import { IsNotEmail } from '@hmcts/cmc-validators/dist/isNotEmail'
```

Comment:
As above, shouldn't need `/dist/isNotEmail` - merge in with the `MaxLength` import

Reply:
When I do that it throws compilation error that IsnotEmail not found in @hmcts/cmc-validators. Am I missing something?

Reply:
Update cmc-validators to 0.2.6. Apologies for my publishing mistake!